packages/terra-tabs/src/ModularTabs.module.scss

```diff
@@ -95,25 +95,40 @@
       }
     }
 
-    .collapsible-tabs-container:focus .is-active,
+    .collapsible-tabs-container:focus .is-active {
+      background-color: var(--terra-tabs-modular-active-focus-background-color, #fff);
+      background-image: var(--terra-tabs-modular-active-focus-background-image, linear-gradient(to bottom, #007ee0, #007ee0));
+      border-radius: var(--terra-tabs-modular-active-focus-border-radius, 0);
+      position: relative;
+      z-index: 1;
+
+      [dir=rtl] & {
+        box-shadow: var(--terra-tabs-keyboard-active-focus-rtl-box-shadow, 0 0 1px 2px #4a90e2);
+      }
+
+      [dir=ltr] & {
+        box-shadow: var(--terra-tabs-keyboard-active-focus-ltr-box-shadow, 0 0 1px 2px #4a90e2);
+      }
+    }
+
     .tab-menu:focus.is-active {
       background-color: var(--terra-tabs-modular-active-focus-background-color, #fff);
-      background-image: var(--terra-tabs-modular-background-image, linear-gradient(to bottom, #007ee0, #007ee0));
-      border-radius: var(--terra-tabs-modular-focus-border-radius, 0);
+      background-image: var(--terra-tabs-modular-active-focus-background-image, linear-gradient(to bottom, #007ee0, #007ee0));
+      border-radius: var(--terra-tabs-modular-active-focus-border-radius, 0);
       position: relative;
       z-index: 1;
 
       [dir=rtl] & {
-        box-shadow: var(--terra-tabs-keyboard-focus-rtl-box-shadow, 0 0 1px 2px #4a90e2);
+        box-shadow: var(--terra-tabs-keyboard-active-focus-rtl-box-shadow, 0 0 1px 2px #4a90e2);
```

Comment:
nit: I'd remove the word `keyboard` from this variable name and the one below it.

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/8a9967b6e0cf7ff56d7f2d657946e5c82e1c59fc#diff-ee126436faa9db1c57ae86d278387a97R128

Reply:
Hmm, it looks like the diff you linked is for `packages/terra-form-textarea/src/Textarea.module.scss` This change needs to go into `packages/terra-tabs/src/ModularTabs.module.scss`

Reply:
ahh. here's the update: https://github.com/cerner/terra-core/pull/2125/commits/f4608e6e9195c0571454c951b2a7c4b0197021e8